### DISCUSSION TITLE
v1.2.0-beta

### DISCUSSION THREAD

**Discussion by tea-artist** [teableio/teable#555] [Announcements]
### Kanban
* feat: Kanban view by @Sky-FE in https://github.com/teableio/teable/pull/528

https://github.com/teableio/teable/assets/113977203/d48d576a-ea9a-4b9e-9db4-2c44c0aa454d

[Kanban view docs](https://help.teable.io/product/views/view-type/kanban-view)

### Import&Export
* feat: support excel import by @caoxing9 in https://github.com/teableio/teable/pull/431
* feat: support increment import by @caoxing9 in https://github.com/teableio/teable/pull/484
* feat: compatible `tsv` import by @caoxing9 in https://github.com/teableio/teable/pull/446
* feat: export csv by @caoxing9 in https://github.com/teableio/teable/pull/493

https://github.com/teableio/teable/assets/113977203/38901878-8631-4fc8-a2bc-22bb49c93671

[Import&Export docs](https://help.teable.io/product/tables/import)

### Auth
* feat: integrate github login by @boris-w in https://github.com/teableio/teable/pull/432
* feat: support google oauth2 by @boris-w in https://github.com/teableio/teable/pull/466

### Search
* feat: search api by @tea-artist in https://github.com/teableio/teable/pull/507
* feat: support search along with link filter by @tea-artist in https://github.com/teableio/teable/pull/517

https://github.com/teableio/teable/assets/113977203/5bd3bddc-b8e6-4b58-93d5-1e7731304fb3

## What's Changed
* feat: support excel import by @caoxing9 in https://github.com/teableio/teable/pull/431
* feat: integrate github login by @boris-w in https://github.com/teableio/teable/pull/432
* chore: remove electron by @tea-artist in https://github.com/teableio/teable/pull/438
* feat: credit check by @tea-artist in https://github.com/teableio/teable/pull/436
* refactor: more appropriate email template structure by @boris-w in https://github.com/teableio/teable/pull/437
* chore: make pnpm automatically install the specified version of Node.js by @dammy001 in https://github.com/teableio/teable/pull/433
* fix: the error of copying and pasting empty content by @boris-w in https://github.com/teableio/teable/pull/443
* feat: compatible `tsv` import by @caoxing9 in https://github.com/teableio/teable/pull/446
* fix: rename link results in illegal data by @tea-artist in https://github.com/teableio/teable/pull/448
* fix: unexpected horizontal lines on login page by @boris-w in https://github.com/teableio/teable/pull/447
* test: import test file save in tmp dir by @caoxing9 in https://github.com/teableio/teable/pull/449
* docs: add deploy on Zeabur in Readme by @MichaelYuhe in https://github.com/teableio/teable/pull/451
* build:  image support `multi-platforms` & app image support `csp-open` by @Pengap in https://github.com/teableio/teable/pull/445
* fix: allow default db address by @tea-artist in https://github.com/teableio/teable/pull/461
* feat: forgot password and setup new password by @boris-w in https://github.com/teableio/teable/pull/452
* fix: execute repeat when import upload by @caoxing9 in https://github.com/teableio/teable/pull/464
* fix: notify jump url & upgrade deps by @Pengap in https://github.com/teableio/teable/pull/465
* fix: ensure interface-provided 'orderBy' parameter overrides view's default sort by @boris-w in https://github.com/teableio/teable/pull/463
* feat: support google oauth2 by @boris-w in https://github.com/teableio/teable/pull/466
* refactor: api for adjusting the order by @tea-artist in https://github.com/teableio/teable/pull/453
* feat: ci add `coverallsapp/github-action` by @Pengap in https://github.com/teableio/teable/pull/468
* fix: vitest `coverage.reporter` config by @Pengap in https://github.com/teableio/teable/pull/469
* chore(example): update dockers examples by @Pengap in https://github.com/teableio/teable/pull/474
* docs: Update header titles for consistency by @codesmith-emmy in https://github.com/teableio/teable/pull/455
* Add one-click deploy on sealos. by @zzjin in https://github.com/teableio/teable/pull/462
* chore: Update constants.ts by @eltociear in https://github.com/teableio/teable/pull/457
* refactor: row order by @tea-artist in https://github.com/teableio/teable/pull/473
* feat: release packages prepare by @boris-w in https://github.com/teableio/teable/pull/477
* docs: fix discord by @tea-artist in https://github.com/teableio/teable/pull/480
* fix: sharing pages set anonymizer user information by default by @boris-w in https://github.com/teableio/teable/pull/481
* fix: drag row order not work by @tea-artist in https://github.com/teableio/teable/pull/482
* fix: Date Picker timezone by @Pengap in https://github.com/teableio/teable/pull/483
* refactor: move zod schema to openapi by @tea-artist in https://github.com/teableio/teable/pull/487
* chore: husky to v9 and upgrade more deps by @Pengap in https://github.com/teableio/teable/pull/494
* chore: add licenses for non-NPM packages by @tea-artist in https://github.com/teableio/teable/pull/495
* fix: unexpected link convert by @tea-artist in https://github.com/teableio/teable/pull/496
* fix: create access token expiration date format by @boris-w in https://github.com/teableio/teable/pull/499
* fix: volumes use host directory permission error by @Pengap in https://github.com/teableio/teable/pull/501
* fix: notify url splicing error by @Pengap in https://github.com/teableio/teable/pull/504
* fix: date field grouping collapse by @Sky-FE in https://github.com/teableio/teable/pull/505
* feat: export csv by @caoxing9 in https://github.com/teableio/teable/pull/493
* fix: filter popover interaction by @caoxing9 in https://github.com/teableio/teable/pull/511
* feat: support increment import by @caoxing9 in https://github.com/teableio/teable/pull/484
* feat: search api by @tea-artist in https://github.com/teableio/teable/pull/507
* fix: useHotkeys ref causes key non-response by @tea-artist in https://github.com/teableio/teable/pull/513
* feat: support search along with link filter by @tea-artist in https://github.com/teableio/teable/pull/517
* fix: search link candidate mismatch by @tea-artist in https://github.com/teableio/teable/pull/521
* fix: popover unexpect disappeared by @tea-artist in https://github.com/teableio/teable/pull/522
* fix: sync ensure upload file dir by @caoxing9 in https://github.com/teableio/teable/pull/524
* fix: link candidate list error by @tea-artist in https://github.com/teableio/teable/pull/531
* fix: form related by @Sky-FE in https://github.com/teableio/teable/pull/535
* refactor: user and link selector by @boris-w in https://github.com/teableio/teable/pull/514
* fix: hide field display in expand records by @boris-w in https://github.com/teableio/teable/pull/537
* fix: copy safari  and optimize some error messages by @boris-w in https://github.com/teableio/teable/pull/542
* feat: Kanban view by @Sky-FE in https://github.com/teableio/teable/pull/528
* fix: date field paste by @boris-w in https://github.com/teableio/teable/pull/545
* fix: invalid single select link in filter by @boris-w in https://github.com/teableio/teable/pull/547
* feat: collaborator cursor support across grid view  by @caoxing9 in https://github.com/teableio/teable/pull/530
* chore: adjusting cookie expiration time for login session by @boris-w in https://github.com/teableio/teable/pull/544
* feat: support import result notify by @caoxing9 in https://github.com/teableio/teable/pull/510
* fix: kanban icon by @Sky-FE in https://github.com/teableio/teable/pull/549
* fix: inplace import field selector overflow can not scroll by @caoxing9 in https://github.com/teableio/teable/pull/551
* fix: field list order may random by @tea-artist in https://github.com/teableio/teable/pull/552
* fix: compatible with old sorting by @tea-artist in https://github.com/teableio/teable/pull/553
* refactor: the hotkeys for the grid by @Sky-FE in https://github.com/teableio/teable/pull/554

## New Contributors
* @dammy001 made their first contribution in https://github.com/teableio/teable/pull/433
* @MichaelYuhe made their first contribution in https://github.com/teableio/teable/pull/451
* @zzjin made their first contribution in https://github.com/teableio/teable/pull/462
* @eltociear made their first contribution in https://github.com/teableio/teable/pull/457

**Full Changelog**: https://github.com/teableio/teable/compare/v1.1.0-beta...v1.2.0-beta

<hr /><em>This discussion was created from the release <a href='https://github.com/teableio/teable/releases/tag/v1.2.0-beta'>v1.2.0-beta</a>.</em>

**Comment by tea-artist** [comment]
Hey everyone! We've got some exciting plans on the horizon and we'd love your input. 🚀 Check out our roadmap and cast your votes on what you'd like to see next. Your feedback is super valuable to us! Let's shape the future together. Check it out here: https://app.teable.io/share/shr04TEw1u9EOQojPmG/view 🌟

**Comment by jf7642132** [comment]
希望能增加在浏览器中直接打开预览附件文档的功能
